jwst/datamodels/mask.py

```diff
@@ -22,7 +23,7 @@ def __init__(self, init=None, **kwargs):
         super(MaskModel, self).__init__(init=init, **kwargs)
 
         if self.dq is not None or self.dq_def is not None:
-            self.dq = dynamic_mask(self)
+            self.dq = dynamic_mask(self, pixel)
```

Comment:
The old  calling sequence of `dynamic_mask()` did not require a specification of which mnemonic map to use, i.e. `pixel` or `group`.  I see the version in `stcal` now does.  This is a breaking change.  I don't see this used outside of `jwst`, `romancal` or `stcal`, so perhaps this is OK?  Or would it be good to have default specified in the version now in `stcal`?

Also, the `jwst/datamodels/dynamicdq.py` module still exists in this package.  We should probably remove it in this PR, or just keep it as if we are now using the version in `stcal`.  It is used in a bunch of our datamodels:

https://github.com/spacetelescope/jwst/search?q=dynamicdq

and of course `romcal` too.

Reply:
I think placing a default `pixel` in stcal may lead to problems, especially if the function is called with the old signature which uses the default `pixel` definitions but the jwst ones have changed. I'd rather have code like that fail and be corrected.

Good point for the `dynamicdq` module in jwst! I missed that. Update coming.

Reply:
Agree!  👍